src/main/java/Task.java

```diff
@@ -0,0 +1,51 @@
+import java.time.format.DateTimeFormatter;
+
+public abstract class Task {
+    String desc;
```

Comment:
I think it is better to spell description fully, so that it is clearer to others.